examples/browser/package.json

```diff
@@ -4,6 +4,10 @@
   "version": "0.0.1-alpha.1",
   "license": "Apache-2.0",
   "dependencies": {
+    "@types/globalize": "0.0.30",
```

Comment:
1. remove `@types/globalize`, `cldr-data`, `globalize`, they are transitively referenced via `theia-core`
2. `globalize-webpack-plugin` should go to `devDependencies`
3.  this file should not be changed manually but be regenerated via `yo theia:browser`, make proper changes in generator-theia
4. the same for electron